### PR TITLE
RenderMeshes query method added

### LINKED ISSUE #927: Add RenderMeshes query method
#### Description:

Missing basic `RenderMeshes` query method, as noticed by @michaelhoehn

### PR DESCRIPTION
**Pull request by pawelbaran**
### Issues addressed by this PR

Closes #927

### Test files

On [SharePoint](https://burohappold.sharepoint.com/sites/BHoM/02_Current/Forms/AllItems.aspx?RootFolder=%2Fsites%2FBHoM%2F02%5FCurrent%2F12%5FScripts%2F01%5FTest%20Scripts%2FRevit%5FToolkit%2FRevit%5FToolkit%2DIssue927%2DRenderMeshes&FolderCTID=0x0120008122C8891F89054B8ACED0196C70DFC4)

### Changelog

- `RenderMeshes` query method added

### CODE REVIEW

File: Revit_Engine/Revit_Engine.csproj [hunk 1]
```diff
@@ -81,6 +81,11 @@
       <HintPath>C:\ProgramData\BHoM\Assemblies\Geometry_oM.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Graphics_oM, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\..\..\..\ProgramData\BHoM\Assemblies\Graphics_oM.dll</HintPath>
```
Comment (al-fisher) [inline Revit_Engine/Revit_Engine.csproj:86]:
```suggestion
      <HintPath>C:\ProgramData\BHoM\Assemblies\Graphics_oM.dll</HintPath>
```
This annoying path slipped through 
😄

Comment (al-fisher) [reply Revit_Engine/Revit_Engine.csproj:86]: @pawelbaran 👆

Comment (pawelbaran) [reply Revit_Engine/Revit_Engine.csproj:86]: Well spotted, thank you @al-fisher! I told myself 'do not forget about changing the reference path'... 😆

**Review by michaelhoehn — APPROVED** [review]
Happy with this method! Tested and works as expected. 
Thanks @pawelbaran

**Comment by michaelhoehn** [comment]
image
Got to love that BHoM branded grass though!

**Comment by pawelbaran** [comment]
Looks like default color is red 😮 I've noted it down in #711, to be investigated.

**Comment by michaelhoehn** [comment]
> Looks like default color is red 😮 I've noted it down in #711, to be investigated.

Seems like a feature to me 🥳

**Pull request merged**